drivers/include/saul.h

```diff
@@ -148,41 +148,76 @@ enum {
  * and the SAUL intra-category ID (six least significant bits).
  */
 enum {
-    SAUL_ACT_ANY            = SAUL_CAT_ACT | SAUL_ACT_ID_ANY,               /**< any actuator - wildcard */
```

Comment:
not sure if we really want to change it this way, maybe its better to relax vera++ settings to allow 120 chars per line before error

Reply:
ah damn coding conventions are strict for 100 char tops, and I don't want to start a lengthy discussion (now)

Reply:
So we leave it as is?

Reply:
yeah for now, maybe we should discuss the 100 char line limit in our coding conventions and argue for 120. IIRC Python has 120 max, bc 80 is nice to have but often a bit to strict.

Reply:
just compiled the doc, still looks good - so let's go for it, regardless of the 100 - 120 chars discussion.

Reply:
the only thing I don't like about this change is that now there are two different styles of documentation for enums in this file.

Reply:
Yeah but I still favor same-line-comments if possible.